src/mme-app/mmeStates/s1HoWfHoNotify.cpp

```diff
@@ -21,8 +21,7 @@
 #include "utils/mmeTimerTypes.h"
 
 #include "mmeStates/s1HoWfHoNotify.h"	
-#include "mmeStates/s1HoWfModifyBearerResponse.h"	
-#include "mmeStates/s1HoWfHoNotify.h"
```

Comment:
New template code is to avoid repeated include files ?

Reply:
Yes..